Auto-focus on load_val when loading data to memory

progrestian opened the pull request:
When inputting a lot of instructions from a table to the simulator, it takes a long time to click the input field manually over and over again.
This proposed change attempts to avoid the issue by auto-focusing on the input field after an accepted input value.

progrestian committed:
Auto-focus on load_val when loading data to memory

When inputting a lot of instructions from a table to the simulator, it takes a long time to click the input field manually over and over again.
This proposed change attempts to avoid the issue by auto-focusing on the input field after an accepted input value.

a1egro commented:
Thanks for the contribution!
Maybe it would be better to use `.select()` instead, could you change that please?

progrestian committed:
Auto-select on load_val when loading data to memory

Also add "select" to element (after focus) to avoid repetitive action.
Previously, one needs to press CTRL-A then BACKSPACE to clear the data, now just BACKSPACE is enough.

Note: .focus() is still needed
Reference: https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/select
